src/Uno.UI/Microsoft/UI/Xaml/Controls/TabView/TabViewItem.cs

```diff
@@ -0,0 +1,406 @@
+﻿using System.Numerics;
+using Uno.UI.Helpers.WinUI;
+using Windows.Devices.Input;
+using Windows.Foundation;
+using Windows.System;
+using Windows.UI.Core;
+using Windows.UI.Input;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Automation;
+using Windows.UI.Xaml.Automation.Peers;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+
+namespace Microsoft.UI.Xaml.Controls
+{
+	public partial class TabViewItem : ListViewItem
+	{
+		private const string c_overlayCornerRadiusKey = "OverlayCornerRadius";
+		private const string SR_TabViewCloseButtonName = "TabViewCloseButtonName";
+
+
+		public TabViewItem()
+		{
+			//__RP_Marker_ClassById(RuntimeProfiler.ProfId_TabViewItem);
+
+			DefaultStyleKey = typeof(TabViewItem);
+
+			SetValue(TabViewTemplateSettingsProperty, new TabViewItemTemplateSettings());
+
+			RegisterPropertyChangedCallback(SelectorItem.IsSelectedProperty, OnIsSelectedPropertyChanged);
+		}
+
+		protected override void OnApplyTemplate()
+		{
+			var popupRadius = (CornerRadius)(ResourceAccessor.ResourceLookup(this, c_overlayCornerRadiusKey));
```

Comment:
Codacy found an issue: [Remove the unused local variable 'popupRadius'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6063676)